tests/testsuite/alt_registry.rs

```diff
@@ -346,7 +346,7 @@ fn publish_with_registry_dependency() {
 See https://doc.rust-lang.org/cargo/reference/manifest.html#package-metadata for more info.
 [PACKAGING] foo v0.0.1 ([ROOT]/foo)
 [UPDATING] `alternative` index
-[PACKAGED] 4 files, [FILE_SIZE]B ([FILE_SIZE]B compressed)
```

Comment:
Was a bit unsure about redactin file count for `cargo package`. Seems they all verify the uploaded content so should be fine